contracts/hardhat.config.cjs

```diff
@@ -153,4 +153,40 @@ task('set-burn-fee-ratio', 'Change the burn fee ratio')
     console.log(`Burn Fee Ratio changed from:${curBurnFeeRatio} to ${newBurnFeeRatio}`)
   })
 
+task('send-klay', 'Send $KLAY from faucet')
+  .addParam('address', 'The receiving address')
+  .addParam('amount', 'The amount of $KLAY to be send')
+  .setAction(async (taskArgs, hre) => {
+    let wallet
+    const PROVIDER = process.env.PROVIDER || ''
+    const provider = new ethers.providers.JsonRpcProvider(PROVIDER)
+
+    if (process.env.PRIVATE_KEY) {
+      const PRIVATE_KEY = process.env.PRIVATE_KEY || ''
+      wallet = await new ethers.Wallet(PRIVATE_KEY, provider)
+    } else {
+      const MNEMONIC = process.env.MNEMONIC || ''
+      wallet = ethers.Wallet.fromMnemonic(MNEMONIC).connect(provider)
+    }
+
+    const to = taskArgs.address || ''
```

Comment:
I think the `|| ''` is not necessary because the command requires to pass `address` parameter. Making it empty string does not solve anything. If it is possible to pass an empty string, we should stop the command from further processing.